ios/RCTTWVideoModule.m

```
@@ -123,6 +123,71 @@ - (void)addLocalView:(TVIVideoView *)view {
   [self updateLocalViewMirroring:view];
 }
 
+- (void)captureLocalFrame:()string {
+  if (self.camera) {
```

Nit: Might be best to fail early instead of nesting this block of code.
```
if (!self.camera) {
   return;
}
```